contrib/devtools/update-translations.py

```diff
@@ -30,6 +30,8 @@
 LOCALE_DIR = 'src/qt/locale'
 # Minimum number of messages for translation to be considered at all
 MIN_NUM_MESSAGES = 10
+# Regular expression for addresses
+ADDRESS_RE = re.compile('(bc1|[13])[a-zA-HJ-NP-Z0-9]{25,34}')
```

Comment:
How about multisig?

Reply:
I thought the address format is same for multisig. Could you provide some info about it if I am mistaken.

Reply:
Sorry it's already covered. Maybe change to `(1|3|bc1)`?

Reply:
done